pandas/core/arrays/_mixins.py

```diff
@@ -474,16 +475,24 @@ def insert(self, loc: int, item) -> Self:
 
     def value_counts(self, dropna: bool = True) -> Series:
         """
-        Return a Series containing counts of unique values.
+        Return a Series containing counts of each unique value.
 
         Parameters
         ----------
         dropna : bool, default True
-            Don't include counts of NA values.
+            Don't include counts of missing values.
 
         Returns
         -------
         Series
+
+        Examples
+        --------
+        >>> arr = pd.array([4, 5])
```

Comment:
this won't give a NDArrayBackedEA

Reply:
The docstrings for the other methods are the EA docstrings, so I've used an example similar to those here. Is there a better way of going about this? I can't see how to write an example using a NDArrayBackedEA without several lines initialising an ExtensionDtype and NDArrayBackedEA